redaxo/src/addons/phpmailer/boot.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
+if (!rex::isBackend() && $this->getConfig('errormailer') == true) {
```

Comment:
Warum nur im frontend?

Reply:
habe ich von @phoebusryan übernommen. Ich sag mal so. Ein Fehler im Backend ist offensichtlich - nicht?

Reply:
@staabm wie ist deine Einstellung dazu?

Reply:
Die mail versendet in der regel ja mails vorhergehender requests, daher passt das argument so vllt nicht.

Dennoch würde ich es denke ich so lassen, damit bei ggf. Error im mailversand nicht das backend KO geht und man nichts mehr machen kann.

Daher finde ich, so lassen 👍

Reply:
😘

Reply:
Anstelle einer Checkbox könnte man auch ein Select anbieten

Systemlog per E-Mail senden:
- nur Frontend
- Frontend und Backend

> bisslaufhübschen

Das wäre dann damit auch erledigt